GSA-TTS/FAC
# Jp/search audit year

# Search - Pre-populate Form, Summary Link, Link Icons, Formatting

Base branch PR: https://github.com/GSA-TTS/FAC/pull/2419

## Changes:
1. Add Audit Year as a searchable field.
    * The relevant form (choices plus field) and view (cast to int, pass to search) updates
    * The HTML template update (loops on form choices for one source of year updates)
    * Test updates in `test_search.py`
2. Add default values of "None" to the parameters in `search_general`
    * Relevant test updates to remove now unnecessary params

## How to Test:
1. Switch to this branch and run normally.
4. Navigate to search and verify that audit year is included in the search params.
5. Make a search (or several) on audit years, and verify that results are as expected. 
6. Verify that the checkboxes are still checked after the search runs.
7. Run the tests for `dissemination.test_search` and verify that things pass. 

## Screenshots:
![image](https://github.com/GSA-TTS/FAC/assets/91098850/438236b3-9ebf-467e-9e80-d97250165365)

## PR checklist: submitters


## PR checklist: reviewers

        
The larger the PR, the stricter we should be about these points.
